components/src/node-main/node-main.js

```diff
@@ -21,16 +21,28 @@ combineDefaults(MathJax.config.loader, 'provides', provides);
 
 MathJax.config.loader.paths.mathjax = (function () {
   //
-  // Try to locate the mathjax-full or mathjax package
-  // If neither is found, try to use the directory where this file is located
+  // Convert a windows path to a unix path (when needed)
+  //
+  const convertWindows = (name) => (path.win32 || !name.match(/^[a-z]:\\/i)) ? name : name.replace(/\\/g, '/');
```

Comment:
I believe the `path.win32` object exists on all platforms, so that will always be true.
Could that not be easier (and maybe more robustly) checked with 
```process.platform === 'win32'```?
For example,
```
process.platform === 'win32' ? path.win32.normalize(name) : name;
```

Reply:
> I believe the `path.win32` object exists on all platforms

Note in the web packed version.  That is the source of the issue.  Webpack apparently provides its own replacement for the `path` library and it is effetely only the posix implementation.  There is no `path.win32` in the web packed components.  This will affect users how install the `mathjax` rather than the `mathjax-full` npm package, as they will be using the web packed version (since the source versions are not available).  In order to test this, you have to make the components and `require('./es5/node-main')` rather than `require('mathjax-full')` (or temporarily change the 'main' entry in the `package.json` file.)

Similarly, the `path.normalize` approach doesn't work in the webpacked version (and `path.win32.normalize()` isn't there) since it only knows about posix directory separators.

I'll have to check whether `process.platform` is correct in webpack (I assume it will be, but the issues with `path` make me worried).

Reply:
Just to be clear, the `path.win32` teat was to determine if we are in the webpacked version or not.  Outside the web packed version, everything works properly in the original code.  It is the webpacked one that is being addressed, here.

Reply:
I did not know that. I always thought webpack works on node and has access to all its built-in libraries. The current version works, so we should leave it as is. I just thought we could short-cut.

Reply:
I was surprised, too.  I had wanted to use `path.win32.normalize` as well, but it wasn't there, and `path.normalize` didn't work.  So I was forced to use this hack.